### PR TITLE
Improve integration tests

### PR DESCRIPTION
**Pull request by meltsufin**
* Stream maven output to console for integration tests
* Make Dev App Server tests more robust

### CODE REVIEW

**Comment by meltsufin** [comment]
@loosebazooka PTAL
I made some improvements to the tests and how they execute on Travis and AppVeyor.

File: src/test/java/com/google/cloud/tools/maven/it/AbstractMojoIntegrationTest.java [hunk 1]
```diff
@@ -31,13 +36,11 @@
 
   private static boolean doneInstallPlugin = false;
 
-  @BeforeClass
+//  @BeforeClass
```
Comment (loosebazooka) [inline src/test/java/com/google/cloud/tools/maven/it/AbstractMojoIntegrationTest.java]: comment introduces some uncertainty, do we want a beforeclass here or no?

Comment (meltsufin) [inline src/test/java/com/google/cloud/tools/maven/it/AbstractMojoIntegrationTest.java]: Great catch! I did this temporarily during testing and forgot to change it back.

File: src/test/java/com/google/cloud/tools/maven/it/AbstractMojoIntegrationTest.java [hunk 2]
```diff
@@ -104,4 +107,53 @@ private static String getUrlContentWithRetries(String url, long timeoutMs, long
 
     return content;
   }
+
+  protected Verifier createStandardVerifier(String testName)
```
Comment (loosebazooka) [inline src/test/java/com/google/cloud/tools/maven/it/AbstractMojoIntegrationTest.java]: I understand this is programming style thing, but would prefer these utilities be in their own classes and called into from the actual tests themselves.

Comment (meltsufin) [inline src/test/java/com/google/cloud/tools/maven/it/AbstractMojoIntegrationTest.java]: I agree, I'll rework it.

**Comment by loosebazooka** [comment]
:ship:

**Pull request merged**